frontend/package.json

```diff
@@ -25,15 +25,23 @@
     "@radix-ui/themes": "^2.0.0",
     "@rainbow-me/rainbowkit": "^2.2.0",
     "@tanstack/react-query": "^5.60.5",
+    "base64-js": "^1.5.1",
+    "browserslist": "^4.27.0",
+    "buffer": "^6.0.3",
+    "caniuse-lite": "^1.0.30001753",
```

Comment:
Why did we add all these dependencies if we're not using them? : )

#ThanksClaude